onnx2kerastl/reshape_layers.py

```diff
@@ -205,20 +205,31 @@ def target_layer(x):
 
             else:
                 input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
-                logger.debug('The first argument is Keras/tf layer. Apply keras.Reshape.')
-                logger.debug('Target shape :')
-                logger.debug(np.int32(input_1[1:]))
-
-                if len(np.int32(input_1[1:])) == 1 and np.int32(input_1[1:])[0] == -1:
-                    logger.debug('The first argument is Keras/tf layer. Apply keras.Flatten.')
-                    flatten = keras.layers.Flatten(name=keras_name)
-                    layers[node_name] = flatten(input_0)
+                input_0_shape = input_0.shape
```

Comment:
can you add a comment regarding why we don't use `tf.reshape(input1)`?, I'm sure we're going to look at this code in the  future and won't understand why

Reply:
I'm not sure what you marked here - but this piece of code remained as is. The only relevant change here was another condition that support the reshape of dynamic tensors:
`                if (input_1 == None).any() and (np.array(input_0_shape) == None).any() and len(input_1) < len(input_0_shape)\
                        and input_1[first_mismatch] == -1: #reshape end`